pkg/controller/resourcequota/resource_quota_controller.go

```diff
@@ -311,7 +315,11 @@ func (rq *ResourceQuotaController) syncResourceQuota(resourceQuota api.ResourceQ
 
 	// there was a change observed by this controller that requires we update quota
 	if dirty {
-		_, err = rq.kubeClient.Core().ResourceQuotas(usage.Namespace).UpdateStatus(&usage)
+		v1Usage := &v1.ResourceQuota{}
```

Comment:
yeah, so i was wondering if sync worked on the internal form, could we then have this block move to a function call that then converted usage to v1Usage and did the update status call?